### PR TITLE
PGOV-963 Update sandbox code and clone Live DB to Dev

### PR DESCRIPTION
**Pull request by kxwang**

### CODE REVIEW

File: .github/workflows/rebase-sandbox-with-master.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  rebase-and-push:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out the Sandbox branch from Github
+        uses: actions/checkout@v3
+        with:
+          ref: sandbox
+
+      - name: Update sandbox on Pantheon
```
Comment (RichardDavies) [inline .github/workflows/rebase-sandbox-with-master.yml:19]: This doesn't technically update anything on Pantheon, does it? It looks like it's just doing everything with the git repo on GitHub. I don't see any problem with it, just that the name is misleading.

**Review by RichardDavies — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -439,7 +439,13 @@ clone_live_database: &clone_live_database
         when: on_success
         no_output_timeout: 45m # allow 45 minutes instead of 10 minute default for script to complete
         command: |
-          echo "export PANTHEON_ENV=$CIRCLE_BRANCH" >> $BASH_ENV && source $BASH_ENV
+          # On Master branch, clone to Dev
+          if [[ $CIRCLE_BRANCH = "master" ]]; then
```
[the remark this answers is not in the document]
Comment (kxwang) [reply .circleci/config.yml:443]: Good catch! Added the workflow condition

File: .github/workflows/rebase-sandbox-with-master.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Github action to rebase sandbox with master
+name: rebase-sandbox-with-master
+
+# Run every Monday at 12:30 UTC (5:30pm Sunday PDT)
```
[the remark this answers is not in the document]
Comment (kxwang) [reply .github/workflows/rebase-sandbox-with-master.yml]: Fix the misleading comment: it should be 00:30

File: .github/workflows/rebase-sandbox-with-master.yml (hunk 1 above)
Comment (kxwang) [reply .github/workflows/rebase-sandbox-with-master.yml]: Fixed the comment: should be rebase on Github

**Review by RichardDavies — APPROVED** [review]

**Pull request merged**